# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gpe-soundserver/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gpe-soundserver-0.4-1/Makefile.vanilla	2007-08-12 16:08:33.000000000 +0200
+++ gpe-soundserver-0.4-1/Makefile	2007-08-12 16:08:48.000000000 +0200
@@ -15,7 +15,7 @@
 endif
 CPPFLAGS += -DPACKAGE=\"$(PACKAGE)\" -DPREFIX=\"$(PREFIX)\" -DPACKAGE_LOCALE_DIR=\"$(PREFIX)/share/locale\"
 
-LDFLAGS += `pkg-config --libs x11`
+LDFLAGS += $(shell pkg-config --libs x11)
 
 OBJS = $(patsubst %,%.o,$(MEMBERS))
 
